main-english comments.py

```diff
@@ -1,180 +1,181 @@
+    width, height = originalImage.size 
+    converted_data_array = np.zeros((height, width, data_Format), dtype=np.uint8) 
+ 
+    for i in range(height): 
+        sortedRow = data_array_sortByRow[i].copy() 
+        quickSortHelp(sortedRow, howBright) 
+        converted_data_array[i] = sortedRow 
+ 
+    converted_image_row = Image.fromarray(converted_data_array, "RGBA"[:data_Format]) 
+    return converted_image_row 
+ 
+ 
+### Convert -> converting each column 
+def convertColumn(originalImage): 
+    print(originalImage) 
+    # to Sort by columns, the easiest way was to just switch the axis of the picture. This way we don't have to create 
+    # a separate quickSort Algorithm. What used to be the columns are now rows, what used to be rows are columns 
+    data_array = switchAxis(np.asarray(originalImage)) 
+    data_array_sortByColumn = data_array.copy() 
+    data_Format = len(data_array[0][0]) 
+    width, height = originalImage.size 
+    converted_data_array = np.zeros((width, height, data_Format), dtype=np.uint8) 
+ 
+    for i in range(width): 
+        sortedColumn = data_array_sortByColumn[i].copy() 
+        quickSortHelp(sortedColumn, howBright) 
+        converted_data_array[i] = sortedColumn 
+ 
+    # when converting the array into an image, we have to remember to switch the axis back the way they were 
+    converted_image_column = Image.fromarray(switchAxis(converted_data_array), "RGBA"[:data_Format]) 
+    return converted_image_column 
+ 
+ 
+# a helpermethod for convertColumn 
+def switchAxis(data_array): 
+    switched_data_array = np.zeros((len(data_array[0]), len(data_array), len(data_array[0][0])), dtype=np.uint8) 
+    for i in range(len(data_array)): 
+        for j in range(len(data_array[i])): 
+            switched_data_array[j][i] = data_array[i][j] 
+    return switched_data_array 
+ 
+ 
+### Convert -> Continuously 
+def convertContinuously(originalImage): 
+    print(originalImage) 
+    data_array = np.asarray(originalImage) 
+    data_Format = len(data_array[0][0]) 
+    width, height = originalImage.size 
+ 
+    # We change the array so that it has new dimensions 
+    # -> before we had a 3d array, now we have a 2d array 
+    # we use .reshape to achieve this 
+    data_array_sortContinuously = data_array.reshape(width * height, data_Format).copy() 
+    quickSortHelp(data_array_sortContinuously, howBright) 
+    # after having sorted the array, we turn it back into a 3d array 
+    converted_data_array = data_array_sortContinuously.reshape(height, width, data_Format) 
+    converted_image_continously = Image.fromarray(converted_data_array, "RGBA"[:data_Format]) 
+    return converted_image_continously 
+ 
+#### Sorting by pixels 
+original = Image.open('Bilder/Beispiel.PNG') 
+ 
+# Make the image smaller while keeping the proportions 
+factor = 0.25 
+width, height = original.size 
+width, height = int(width * factor), int(height * factor) 
+original = original.resize((width, height)) 
```

Comment:
I would wrap this into a function or move it to just before line 180. It can be confusing to mix function definition and actual scripting.

```suggestion
def load_img(path, scale)
    original = Image.open(path) 
 
    # Make the image smaller while keeping the proportions 
    width, height = original.size 
    new_size = int(width * scale), int(height * scale) 
    original = original.resize(new_size) 
    return original
```